examples/webpack.config.js

```diff
@@ -5,21 +5,58 @@ const CircularDependencyPlugin = require('circular-dependency-plugin');
 const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
 const CompressionPlugin = require('compression-webpack-plugin');
 const webpack = require('webpack');
-
-const title = require('./package.json').description;
+const getConfig = require('./server/src/utils/getConfig');
 
 process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
 
-const defaultBackend = process.env.CLIENT_DEMO_BACKEND || 'https://staging3.intgdc.com';
+const backendShortcuts = {
```

Comment:
Isn't this kind of leaking all the internal server names to public?

Reply:
It doesnt matter - these are only accessible with VPN.